applications/MeshingApplication/custom_strategies/custom_convergencecriterias/error_mesh_criteria.h

```diff
@@ -60,98 +53,87 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
-/** @brief Custom convergence criteria for the mortar condition 
+/**
+ * @class ErrorMeshCriteria
+ * @ingroup MeshingApplication
+ * @brief Custom convergence criteria for the mortar condition
+ * @tparam TSparseSpace The sparse space considered
+ * @tparam TDenseSpace The dense space considered
+ * @author Anna Rehr
+ * @author Vicente Mataix Ferrandiz
  */
 template<class TSparseSpace, class TDenseSpace>
-class ErrorMeshCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpace >
+class ErrorMeshCriteria
+    : public ConvergenceCriteria< TSparseSpace, TDenseSpace >
 {
 public:
     ///@name Type Definitions
     ///@{
 
     KRATOS_CLASS_POINTER_DEFINITION( ErrorMeshCriteria );
 
-    typedef ConvergenceCriteria< TSparseSpace, TDenseSpace > BaseType;
+    typedef ConvergenceCriteria< TSparseSpace, TDenseSpace >        BaseType;
 
-    typedef TSparseSpace                              SparseSpaceType;
+    typedef TSparseSpace                                     SparseSpaceType;
 
-    typedef typename BaseType::TDataType                    TDataType;
+    typedef typename BaseType::TDataType                           TDataType;
 
-    typedef typename BaseType::DofsArrayType            DofsArrayType;
+    typedef typename BaseType::DofsArrayType                   DofsArrayType;
 
-    typedef typename BaseType::TSystemMatrixType    TSystemMatrixType;
+    typedef typename BaseType::TSystemMatrixType           TSystemMatrixType;
 
-    typedef typename BaseType::TSystemVectorType    TSystemVectorType;
+    typedef typename BaseType::TSystemVectorType           TSystemVectorType;
+    
+    typedef ModelPart::ConditionsContainerType           ConditionsArrayType;
     
-    typedef ModelPart::ConditionsContainerType    ConditionsArrayType;
+    typedef ModelPart::NodesContainerType                     NodesArrayType;
     
-    typedef ModelPart::NodesContainerType              NodesArrayType;
+    typedef std::size_t                                              KeyType;
     
-    typedef std::size_t KeyType;
+    typedef std::size_t                                             SizeType;
+
+    ///@}
+    ///@name Enum's
+    ///@{
 
     ///@}
     ///@name Life Cycle
     ///@{
     
     /// Default constructors
-    ErrorMeshCriteria(
-        ModelPart& rThisModelPart,
-        Parameters ThisParameters = Parameters(R"({})")
-        )
+    ErrorMeshCriteria(Parameters ThisParameters = Parameters(R"({})"))
         : ConvergenceCriteria< TSparseSpace, TDenseSpace >(),
-          mThisModelPart(rThisModelPart),
-          mDimension(rThisModelPart.GetProcessInfo()[DOMAIN_SIZE]),
-          mThisParameters(ThisParameters),
-          mFindNodalH(FindNodalHProcess(mThisModelPart))
+          mThisParameters(ThisParameters)
     {
-        Parameters DefaultParameters = Parameters(R"(
+        Parameters default_parameters = Parameters(R"(
```

Comment:
I suggest adding some docu to these parameters, imo they are not very descriptive